types/node/test/globals.ts

```diff
@@ -46,10 +46,21 @@ declare var RANDOM_GLOBAL_VARIABLE: true;
     readonlyArray.at(-1);
 }
 
+// AbortController and AbortSignal
 {
     const x = new AbortController().signal;
     x.reason; // $ExpectType any
     x.throwIfAborted(); // $ExpectType void
+    // @ts-expect-error
+    AbortSignal.any();
+    // @ts-expect-error
+    AbortSignal.any(null);
+    // @ts-expect-error
+    AbortSignal.any(undefined);
+    // @ts-expect-error
+    AbortSignal.any([] as number[]);
+    AbortSignal.any([]); // $ExpectType AbortSignal
+    AbortSignal.any([new AbortController().signal]); // $ExpectType AbortSignal
```

Comment:
seems to be broken since types lack in https://github.com/microsoft/TypeScript/blob/d027e9619fb8ca964df3885a536a67b5f813738b/src/lib/dom.generated.d.ts#L2319-L2326